kratos/tests/cpp_tests/includes/test_property_accessor.cpp

```diff
@@ -90,5 +91,81 @@ KRATOS_TEST_CASE_IN_SUITE(PropertyAccessorSimpleProperties, KratosCoreFastSuite)
     KRATOS_CHECK_NEAR(2.1e11 * 2.0, modified_E_from_acc, 1.0e-8);
 }
 
+/**
+* Checks the correct work of the TableAccessor
+*/
+KRATOS_TEST_CASE_IN_SUITE(TableAccessorSimpleProperties, KratosCoreFastSuite)
```

Comment:
please also add the test with serialization

Reply:
[Sure](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/tests/cpp_tests/sources/test_serializer.cpp)

Reply:
Hi @philbucher I've changed all your comments but I've got a problem. I cannot serialize the `Variable<double>*` so I am currently serializing the `mpVariable->Key()`. This is ok but when loading the `TableAccessor ` the mpVariable is empty. How can I initialize the `mpVariable `according to the loaded `Variable.Key()`?

Reply:
Hm not sure the Key solution is possible

What is the error with serializing the Var directly?

Reply:
I get a SegFault, the member variable is a `Variable<double>*`

Reply:
in the test:
~~~cpp
{
    StreamSerializer serializer;
    TableAccessor table_accessor = TableAccessor(&TEMPERATURE, "node_historical");

    serializer.save("table_accessor_info", table_accessor);

    TableAccessor table_accessor_loaded;

    // Variable<double> *p_var_loaded;
    serializer.load("table_accessor_info", table_accessor_loaded);

    auto& d = table_accessor_loaded.GetInputVariable();

    KRATOS_WATCH(d.Key()) // here is 0
    KRATOS_WATCH(TEMPERATURE.Key()) // here is correct
}
~~~

Reply:
Thanks for the explanation, I need to think

Btw using the name and the retrieving from KratosComponents should work as a workaround

Reply:
The thing is that, the `table_accessor_loaded` uses the default constructor and this one does not initialize the `Variable*`. I am afraid that this is the problem

Reply:
THX for the hint @philbucher, now it is working properly